test/unit/controllers/ChoiceControllerSpec.scala

```diff
@@ -44,6 +50,11 @@ class ChoiceControllerSpec extends ControllerSpec with ErrorHandlerMocks {
     authorizedUser()
   }
 
+  override def postRequest(body: JsValue): Request[AnyContentAsJson] =
```

Comment:
This method is inside `ControllerSpec`

Path: `test/unit/base/ControllerSpec.scala`

Reply:
I wanted a version that did not set a value for decId on the session.

Reply:
Ah, okay, sorry.

What do you think about moving this to `ControllerSpec`? Can it be useful or it'll be just part of this spec?

What do you think about changing name of this method and use it without `override`?

Reply:
I could do either.   I don't think many other controllers would need it as they are mostly dealing with requests involving existing draft decs.   Maybe a more useful name would be a good idea.

Reply:
It's an exceptional circumstance so I'd just create the request manually, but upto you